tools/profile_apply

```diff
@@ -440,13 +442,18 @@ def apply(tabs):
                     label = row['label']
                     field_id = get_field_for_section(label + '[binary]', label)
                     section_id = add_field_to_form(form_id, field_id, None, fn)
-                elif last_type == 'select_multiple':
+                elif last_type == 'select_multiple' or begining:
                     # Fields after the end of a toggle group don't belong in
                     # the section, so put them in a new section, using
                     # "[invisible]" in its name to hide the section heading.
+                    #
+                    # If begining is True then there is no section declared in
+                    # the begining of the form so we also need to create at least
+                    # one "[invisible]" section to avoid chart rendering problems.
```

Comment:
Can you change this to `one "[invisible]" section to work around a bug in the ODK Android library.` ?